mapper/thin_edge_json/src/measurement.rs

```diff
@@ -164,19 +62,19 @@ pub trait FlatMeasurementVisitor {
 ///     }
 /// }
 /// ```
-pub trait GroupedMeasurementVisitor {
-    /// Error type specific to this way of collecting measurements
+pub trait MeasurementVisitor {
```

Comment:
Why not an helper method that combines the sequence start / measurement / end.

Reply:
We can add that to `MeasurementVisitorExt`.